### ISSUE TITLE
OS X installation problem

### ISSUE THREAD

**Issue by mibori** [nachoparker/dutree#5]
```
$ cargo install dutree
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading dutree v0.2.5
  Installing dutree v0.2.5
 Downloading getopts v0.2.17
 Downloading regex v0.2.11
 Downloading terminal_size v0.1.7
 Downloading unicode-segmentation v1.2.1
 Downloading unicode-width v0.1.4
 Downloading thread_local v0.3.5
 Downloading memchr v2.0.1
 Downloading utf8-ranges v1.0.0
 Downloading aho-corasick v0.6.4
 Downloading regex-syntax v0.5.6
 Downloading unreachable v1.0.0
 Downloading lazy_static v1.0.0
 Downloading void v1.0.2
 Downloading libc v0.2.40
 Downloading ucd-util v0.1.1
 Downloading winapi v0.2.8
 Downloading kernel32-sys v0.2.2
 Downloading winapi-build v0.1.1
   Compiling winapi v0.2.8
   Compiling getopts v0.2.17
   Compiling utf8-ranges v1.0.0
   Compiling regex v0.2.11
   Compiling unicode-width v0.1.4
   Compiling winapi-build v0.1.1
   Compiling libc v0.2.40
   Compiling void v1.0.2
   Compiling ucd-util v0.1.1
   Compiling unicode-segmentation v1.2.1
   Compiling lazy_static v1.0.0
   Compiling unreachable v1.0.0
   Compiling kernel32-sys v0.2.2
   Compiling memchr v2.0.1
   Compiling regex-syntax v0.5.6
   Compiling thread_local v0.3.5
   Compiling aho-corasick v0.6.4
   Compiling terminal_size v0.1.7
   Compiling dutree v0.2.5
error[E0433]: failed to resolve. Could not find `linux` in `os`
  --> src/lib.rs:55:14
   |
55 | use std::os::linux::fs::MetadataExt;
   |              ^^^^^ Could not find `linux` in `os`

error[E0599]: no method named `st_blocks` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:230:50
    |
230 |         Ok(metadata) => if usage_flag { metadata.st_blocks()*512 } else { metadata.st_size() },
    |                                                  ^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error[E0599]: no method named `st_size` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:230:84
    |
230 |         Ok(metadata) => if usage_flag { metadata.st_blocks()*512 } else { metadata.st_size() },
    |                                                                                    ^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error[E0599]: no method named `st_mode` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:483:38
    |
483 |         let mode = metadata.unwrap().st_mode();
    |                                      ^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error: aborting due to 4 previous errors

error: failed to compile `dutree v0.2.5`, intermediate artifacts can be found at `/var/folders/28/yf_8fv7s6694pyt259fdlc5m0000gn/T/cargo-install.osYK5NbwvVEz`

Caused by:
  Could not compile `dutree`.

To learn more, run the command again with --verbose.
```
On mac OS High Sierra
```
$ uname -a
Darwin mini500.local 17.5.0 Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64 x86_64
$ rustc --version
rustc 1.25.0
$ cargo --version
cargo 0.25.0
```

**Comment by mibori** [comment]
thnx, but unfortunately

```
$ cargo install dutree
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading dutree v0.2.6
  Installing dutree v0.2.6
   Compiling lazy_static v1.0.0
   Compiling unicode-width v0.1.4
   Compiling regex v0.2.11
   Compiling winapi-build v0.1.1
   Compiling ucd-util v0.1.1
   Compiling utf8-ranges v1.0.0
   Compiling unicode-segmentation v1.2.1
   Compiling libc v0.2.40
   Compiling getopts v0.2.17
   Compiling winapi v0.2.8
   Compiling void v1.0.2
   Compiling kernel32-sys v0.2.2
   Compiling regex-syntax v0.5.6
   Compiling memchr v2.0.1
   Compiling unreachable v1.0.0
   Compiling thread_local v0.3.5
   Compiling aho-corasick v0.6.4
   Compiling terminal_size v0.1.7
   Compiling dutree v0.2.6
error[E0433]: failed to resolve. Could not find `linux` in `os`
  --> src/lib.rs:55:14
   |
55 | use std::os::linux::fs::MetadataExt;
   |              ^^^^^ Could not find `linux` in `os`

error[E0599]: no method named `st_blocks` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:230:50
    |
230 |         Ok(metadata) => if usage_flag { metadata.st_blocks()*512 } else { metadata.st_size() },
    |                                                  ^^^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error[E0599]: no method named `st_size` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:230:84
    |
230 |         Ok(metadata) => if usage_flag { metadata.st_blocks()*512 } else { metadata.st_size() },
    |                                                                                    ^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error[E0599]: no method named `st_mode` found for type `std::fs::Metadata` in the current scope
   --> src/lib.rs:483:38
    |
483 |         let mode = metadata.unwrap().st_mode();
    |                                      ^^^^^^^
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope, perhaps add a `use` for it:
    |
38  | use std::os::macos::fs::MetadataExt;
    |

error: aborting due to 4 previous errors

error: failed to compile `dutree v0.2.6`, intermediate artifacts can be found at `/var/folders/28/yf_8fv7s6694pyt259fdlc5m0000gn/T/cargo-install.PY8yswM2LdAP`

Caused by:
  Could not compile `dutree`.

To learn more, run the command again with --verbose.
```

**Comment by nachoparker** [comment]
Oops, you are right. It seems like I destroyed mac compatibility. Should be fixed now in `v0.2.7`
